lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,36 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance, id: :db_instance_identifier
+
+  attribute :db_instance_identifier, kind_of: String, name_attribute: true
+
+  attribute :engine, kind_of: String
+  attribute :engine_version, kind_of: String
+  attribute :db_instance_class, kind_of: String
+  attribute :multi_az, default: false, kind_of: [TrueClass, FalseClass]
+  attribute :allocated_storage, kind_of: Integer, default: 5
```

Comment:
A common pattern we see with chef-provisioning is someone declaring a resource with all the possible attributes (like `allocated_storage 5000`) and then updating just one resource on the attribute later (like `description "new description"`).

If we default the storage to 5 then when they have their second resource invocation later, it will reset the storage from 5000 down to 5.  This is also a consequence of the fact that chef-provisioning-aws resources don't perform resource cloning.

I'll remove this in the commit I am working on